cfme/tests/services/test_service_catalogs.py

```diff
@@ -202,3 +204,35 @@ def test_advanced_search_registry_element(request, appliance):
     request.addfinalizer(view.search.close_advanced_search)
     view.search.advanced_search_form.search_exp_editor.registry_form_view.fill({'type': "Registry"})
     assert view.search.advanced_search_form.search_exp_editor.registry_form_view.type.is_displayed
+
+
+@pytest.mark.tier(2)
+@pytest.mark.provider([RHEVMProvider], selector=ONE)
+def test_order_service_after_deleting_provider(appliance, provider, setup_provider, catalog_item):
+    """
+    Polarion:
+        assignee: nansari
+        casecomponent: Services
+        testtype: functional
+        initialEstimate: 1/4h
+        startsin: 5.8
```

Comment:
Do we care about 5.8 anymore? Would it make sense to say 5.9 or 5.10 even? Just a thought.

Reply:
I think better to put exact if we know.